tests/testthat/test-utils-fetch.R

```diff
@@ -409,30 +400,28 @@ test_that("fetchPrimarySeriesApprovals properly retrieves the primary series app
   primary <- repgen:::fetchPrimarySeriesApprovals(approvals)
 
   expect_is(approvals, 'list')
-  expect_equal(approvals[[1]][['level']], 0)
-  expect_equal(approvals[[1]][['description']], 'Working')
+  expect_equal(approvals[[1]][['approvalLevel']], 0)
+  expect_equal(approvals[[1]][['levelDescription']], 'Working')
 })
 
 test_that("fetchPrimarySeriesQualifiers properly retrieves the primary series qualifiers", {
-  approvals <- fromJSON('{
+  qualifiers <- fromJSON('{
     "primarySeriesQualifiers": [
         {
-            "startDate": "2016-12-01T00:00:00-05:00",
-            "endDate": "2017-01-10T00:00:00.0000001-05:00",
+            "startTime": "2016-12-01T00:00:00-05:00",
+            "endTime": "2017-01-10T00:00:00.0000001-05:00",
             "identifier": "ESTIMATED",
-            "code": "E",
-            "displayName": "Flow at station affected by ice",
-            "appliedBy": "admin",
+            "user": "admin",
             "dateApplied": "2016-12-10T11:16:12Z"
         }
     ]
   }')
   
-  primary <- repgen:::fetchPrimarySeriesQualifiers(approvals)
+  primary <- repgen:::fetchPrimarySeriesQualifiers(qualifiers)
   
-  expect_is(approvals, 'list')
-  expect_equal(approvals[[1]][['code']], "E")
-  expect_equal(approvals[[1]][['displayName']], "Flow at station affected by ice")
+  expect_is(qualifiers, 'list')
+  expect_equal(qualifiers[[1]][['identifier']], "ESTIMATED")
```

Comment:
had to change these tests because the data for code and displayName was no longer a part of the primarySeriesQualifiers fetch when those data moved to qualifierMetadata